plugins/webhooks/build.js

```diff
@@ -42,10 +42,13 @@ module.exports = () => ({
                 if (['SUCCESS', 'FAILURE', 'ABORTED'].indexOf(status) > -1) {
                     build.meta = request.payload.meta || {};
                     build.endTime = (new Date()).toISOString();
+                } else {
+                    build.startTime = (new Date()).toISOString();
```

Comment:
Not sure how the launcher interacts with this. Does it only hit this endpoint when the on start and notify? Are we sure it won't do it for status = `RUNNING`?